cid/builtin/core/data/resources.yaml

```diff
@@ -123,26 +123,36 @@ datasets:
     dependsOn:
       views:
         - kpi_ebs_snap
+      schedules:
+        - default
   kpi_ebs_storage_all:
     File: kpi/kpi_ebs_storage_all.json
     dependsOn:
       views:
        - kpi_ebs_storage_all
+      schedules:
+        - default
   kpi_instance_all:
     File: kpi/kpi_instance_all.json
     dependsOn:
       views:
         - kpi_instance_all
+      schedules:
+        - default
   kpi_s3_storage_all:
     File: kpi/kpi_s3_storage_all.json
     dependsOn:
       views:
         - kpi_s3_storage_all
+      schedules:
+        - default
   kpi_tracker:
     File: kpi/kpi_tracker.json
     dependsOn:
       views:
         - kpi_tracker
+      schedules:
+        - default
```

Comment:
Should be schedule on the same level as dependsOn ?

Reply:
That's actually a good point as it is not a dependency.